packages/runtime/src/useCases/common/validation/TokenAndTemplateCreationValidator.ts

```diff
@@ -41,6 +42,26 @@ export class TokenAndTemplateCreationValidator<
                     );
                 }
             }
+
+            if (passwordProtection.passwordLocationIndicator) {
+                if (
+                    (typeof passwordProtection.passwordLocationIndicator === "string" &&
+                        (!Object.values(PasswordLocationIndicatorMedium).includes(passwordProtection.passwordLocationIndicator as PasswordLocationIndicatorMedium) ||
```

Comment:
I think if you use Object.keys instead of Object.values you don't need the typecast.